index.js

```diff
@@ -1,126 +1,195 @@
-function removeToDo(e) {
-  e.target.parentNode.remove();
+function generateId() {
+  let id = 0;
+  return function () {
+    return id++;
+  };
 }
 
-function changeTextColor(e) {
-  let newText = e.target.parentNode.querySelector("p");
-  let parent = e.target.parentNode;
-  newText.style.color = "green";
-  newText.classList.add("newText");
-  parent.classList.add("doneToDo");
-  iftrue = true;
+function getTodos() {
+  const todos = localStorage.getItem('todos');
+  return JSON.parse(todos) || [];
 }
-function editToDo(e) {
-  let paragraph = e.target.parentNode.querySelector("p");
-  let newinput = document.createElement("input");
-  console.log(paragraph);
-  newinput.value = paragraph.textContent;
-  paragraph.remove();
-
-  e.target.parentNode.appendChild(newinput);
-  let saveTodo = document.createElement("button");
-  saveTodo.textContent = "Save";
-  saveTodo.classList.add("toDoListbuttons");
-  e.target.parentNode.appendChild(saveTodo);
-  saveTodo.addEventListener("click", saveToDo);
+
+const getID = generateId();
+let todos = getTodos();
+
+function render(todos) {
+
+  console.log(todos)
+  let display = document.querySelector(".display");
+  display.innerHTML = '';
+
+  for (let i = 0; i < todos.length; i++) {
+    let newDiv = document.createElement("div");
```

Comment:
variable name should represent the thing which is contained in it. For example const todo =